apps/health_campaign_field_worker_app/lib/utils/background_service.dart

```diff
@@ -72,7 +72,7 @@ Future<void> initializeService(dio, isar) async {
       autoStart: false,
       isForegroundMode: true,
       initialNotificationContent:
-          'BackGround Service Started at ${DateTime.now()}',
+      'BackGround Service Started at ${DateTime.now()}',
```

Comment:
Please translate the code from the i18 constant.

Reply:
Background service is not a widget , so i am unable to get context so as to use localization